packages/preview-middleware/test/fixtures/simple-app/package-lock.json

```diff
@@ -0,0 +1,12989 @@
+{
```

Comment:
Please remove this file, it doesn't make sense to have it in the test data

Reply:
Thanks, done